tests/field/example-django.sh

```diff
@@ -6,6 +6,14 @@
 # The periodic regeneration is configured as a scheduled
 # pipeline in GitLab > CI/CD > Schedules.
 #
+# To run this field test locally:
+#  (1) Generate a Personal Access Token on GitLab 
+#      Top-right usesr menu > Settings > Access Tokens
+#  (2) export GITLAB_API_TOKEN=<your personal access token>
+#  (3) run ./tests/field/example-django.sh
+# The generated files are found in /tmp/painless-generated-project
+# 
+
```

Comment:
I don't like this empty line. :smirk:

It gives the `set` command more importance than it deserves. (I know that's stupid as a reason, so feel free to object.)